### ISSUE TITLE
[Calendar] Multilanguage cognitive model deployment errors

### ISSUE THREAD

**Issue by ryanisgrig** [microsoft/botframework-solutions#2785]
#### What project is affected?
Calendar Skill
#### What language is this in?
Deployment scripts
#### What happens?
There are various errors when deploying multiple languages.

* Token recognition errors (highlighted below)
* German deployment error: `Labels cannot have two overlapping children of the same parent.`
* French deployment error: `Labels cannot have two overlapping children of the same parent.`
* Italian deployment error: `Labels cannot have two overlapping children of the same parent.`
* Chinese deployment error: `CanonicalForm cannot be null or empty.`

#### What are the steps to reproduce this issue?
Run the deploy scripts on a new Calendar Skill

#### What were you expecting to happen?
All language apps deploy successfully 

#### Can you share any logs, error output, etc.?
**German**
```
> Parsing Calendar LU file ...
line 1211:20 token recognition error at: 'ü'
line 1231:19 token recognition error at: 'ö'
line 1251:21 token recognition error at: 'ß'
line 1255:21 token recognition error at: 'ß'
line 1257:21 token recognition error at: 'ß'
line 1287:22 token recognition error at: 'ä'
```

**French**
```
> Parsing Calendar LU file ...
line 1192:20 token recognition error at: 'è'
line 1194:25 token recognition error at: 'è'
line 1196:20 token recognition error at: 'è'
line 1202:22 token recognition error at: 'è'
line 1204:22 token recognition error at: 'è'
line 1224:19 token recognition error at: 'è'
line 1226:24 token recognition error at: 'è'
line 1228:22 token recognition error at: '('
line 1228:24 token recognition error at: ')'
line 1238:25 token recognition error at: 'è'
line 1242:25 token recognition error at: 'è'
line 1258:19 token recognition error at: 'è'
line 1260:25 token recognition error at: 'è'
line 1268:20 token recognition error at: 'è'
line 1276:20 token recognition error at: 'è'
line 1276:29 token recognition error at: 'œ'
line 1280:25 token recognition error at: 'œ'
line 1282:28 token recognition error at: 'œ'
line 1284:19 token recognition error at: 'œ'
line 1292:28 token recognition error at: '’'
line 1292:29 token recognition error at: 'é'
line 1298:18 token recognition error at: 'é'
```

**Italian**
```
> Parsing Calendar LU file ...
line 1203:23 token recognition error at: '''
```

**Chinese**
```
> Parsing Calendar LU file ...
line 1121:21 token recognition error at: '个'
line 1121:22 token recognition error at: '人'
line 1123:21 token recognition error at: '家'
line 1123:22 token recognition error at: '庭'
line 1126:21 token recognition error at: '工'
line 1126:22 token recognition error at: '作'
line 1128:21 token recognition error at: '微'
line 1128:22 token recognition error at: '软'
line 1132:21 token recognition error at: '网'
line 1132:22 token recognition error at: '易'
line 1136:21 token recognition error at: '谷'
line 1136:22 token recognition error at: '歌'
line 1140:21 token recognition error at: '雅'
line 1140:22 token recognition error at: '虎'
line 1143:21 token recognition error at: '谷'
line 1143:23 token recognition error at: '歌'
line 1149:18 token recognition error at: '丈'
line 1149:19 token recognition error at: '夫'
line 1152:18 token recognition error at: '主'
line 1152:19 token recognition error at: '任'
line 1154:18 token recognition error at: '伯'
line 1154:19 token recognition error at: '伯'
line 1156:18 token recognition error at: '侄'
line 1156:19 token recognition error at: '女'
line 1158:18 token recognition error at: '侄'
line 1158:19 token recognition error at: '子'
line 1160:18 token recognition error at: '儿'
line 1160:19 token recognition error at: '媳'
line 1162:18 token recognition error at: '儿'
line 1162:19 token recognition error at: '子'
line 1164:18 token recognition error at: '公'
line 1164:19 token recognition error at: '公'
line 1166:18 token recognition error at: '同'
line 1166:19 token recognition error at: '学'
line 1168:18 token recognition error at: '哥'
line 1168:19 token recognition error at: '哥'
line 1173:18 token recognition error at: '堂'
line 1173:19 token recognition error at: '姐'
line 1175:18 token recognition error at: '外'
line 1175:19 token recognition error at: '公'
line 1177:18 token recognition error at: '外'
line 1177:19 token recognition error at: '婆'
line 1179:18 token recognition error at: '外'
line 1179:19 token recognition error at: '孙'
line 1179:20 token recognition error at: '女'
line 1181:18 token recognition error at: '外'
line 1181:19 token recognition error at: '孙'
line 1181:20 token recognition error at: '子'
line 1183:18 token recognition error at: '外'
line 1183:19 token recognition error at: '甥'
line 1185:18 token recognition error at: '大'
line 1185:19 token recognition error at: '伯'
line 1187:18 token recognition error at: '女'
line 1187:19 token recognition error at: '儿'
line 1189:18 token recognition error at: '奶'
line 1189:19 token recognition error at: '奶'
line 1191:18 token recognition error at: '妈'
line 1191:19 token recognition error at: '妈'
line 1198:18 token recognition error at: '妹'
line 1198:19 token recognition error at: '妹'
line 1200:18 token recognition error at: '妻'
line 1200:19 token recognition error at: '子'
line 1203:18 token recognition error at: '姐'
line 1203:19 token recognition error at: '夫'
line 1205:18 token recognition error at: '姐'
line 1205:19 token recognition error at: '姐'
line 1210:18 token recognition error at: '姑'
line 1210:19 token recognition error at: '妈'
line 1212:18 token recognition error at: '姑'
line 1212:19 token recognition error at: '姑'
line 1214:18 token recognition error at: '姑'
line 1214:19 token recognition error at: '父'
line 1216:18 token recognition error at: '姨'
line 1216:19 token recognition error at: '妈'
line 1218:18 token recognition error at: '婶'
line 1218:19 token recognition error at: '婶'
line 1220:18 token recognition error at: '媳'
line 1220:19 token recognition error at: '妇'
line 1222:18 token recognition error at: '嫂'
line 1222:19 token recognition error at: '子'
line 1224:18 token recognition error at: '孙'
line 1224:19 token recognition error at: '女'
line 1226:18 token recognition error at: '孙'
line 1226:19 token recognition error at: '子'
line 1228:18 token recognition error at: '小'
line 1228:19 token recognition error at: '姨'
line 1230:18 token recognition error at: '岳'
line 1230:19 token recognition error at: '母'
line 1232:18 token recognition error at: '岳'
line 1232:19 token recognition error at: '父'
line 1234:18 token recognition error at: '弟'
line 1237:18 token recognition error at: '爱'
line 1237:19 token recognition error at: '人'
line 1239:18 token recognition error at: '爷'
line 1239:19 token recognition error at: '爷'
line 1241:18 token recognition error at: '爸'
line 1241:19 token recognition error at: '爸'
line 1248:18 token recognition error at: '老'
line 1248:19 token recognition error at: '师'
line 1250:18 token recognition error at: '舅'
line 1250:19 token recognition error at: '妈'
line 1252:18 token recognition error at: '舅'
line 1252:19 token recognition error at: '母'
line 1254:18 token recognition error at: '舅'
line 1254:19 token recognition error at: '舅'
line 1256:18 token recognition error at: '表'
line 1256:19 token recognition error at: '哥'
line 1258:18 token recognition error at: '表'
line 1258:19 token recognition error at: '妹'
line 1260:18 token recognition error at: '表'
line 1260:19 token recognition error at: '姐'
line 1262:18 token recognition error at: '表'
line 1262:19 token recognition error at: '弟'
line 1264:18 token recognition error at: '钱'
line 1264:20 token recognition error at: '包'
line 1264:22 token recognition error at: '小'
line 1264:24 token recognition error at: '组'
line 1264:26 token recognition error at: '会'
line 1264:28 token recognition error at: '议'
```

**Comment by bobokids** [comment]
the token recognition issue: not repro in ludown v1.3.2. repro in ludown v1.4 and BF CLI. log a bug to BF CLI https://github.com/microsoft/botframework-cli/issues/412

**Comment by bobokids** [comment]
Run with BF CLI 4.7, only the below error repro, other issues were fixed in 4.7. 
[ERROR] line 1174:0 - line 1174:20: PREBUILT entity 'personName' is not available for the requested locale 'de-de'.

The issue is tracked by microsoft/botframework-cli#412, which is tagged as RS8 task.
